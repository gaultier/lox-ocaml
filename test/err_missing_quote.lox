// expect: 3:1:Missing closing quote, no more tokens for string: `abc;`
"hello" + "abc;
