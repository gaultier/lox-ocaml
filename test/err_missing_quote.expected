2:1:Missing closing quote, no more tokens for string: `abc;
